tests/functional/oauth_iframe.js

```diff
@@ -252,7 +252,7 @@ define([
         .end();
     },
 
-    'password reset, verify same browser': function () {
```

Comment:
I changed all of the `password reset`=>`reset password` because we had a mixture of the two and it made running the tests with grep=`..` a pain.